### PR TITLE
Mikhailov/specs parser

### PR DESCRIPTION
**Pull request by cheshirsky**

### CODE REVIEW

**Comment by operatino** [comment]
On main page open, right after app start:

```
[2015-03-03 18:24:00.371] [INFO] app - Started connection on mongodb://localhost/test, waiting for it to open...
[2015-03-03 18:24:00.381] [INFO] app - [SOURCEJS] launched on http://localhost:8080 in development mode...
[2015-03-03 18:24:00.386] [INFO] app - Could not connect to mongo server!
[2015-03-03 18:24:00.386] [INFO] app - failed to connect to [localhost:27017]
[2015-03-03 18:24:00.387] [INFO] app - Spec dependencies JSON saved to /Users/robert/Dropbox/okp/user/data/spec_dependencies_tree.json
[2015-03-03 18:24:00.389] [INFO] app - Pages tree JSON saved to /Users/robert/Dropbox/okp/core/api/data/pages-tree.json
[2015-03-03 18:24:11.284] [WARN] app - Parse Data: Api data of specs does not exist, please update HTML API data.
[2015-03-03 18:24:11.285] [WARN] app - Error:  [SyntaxError: Unexpected end of input]
SyntaxError: Unexpected end of input
    at Object.parse (native)
    at ParseData.getRaw (/Users/robert/Dropbox/okp/core/api/parseData.js:84:21)
    at parseData.scope (/Users/robert/Dropbox/okp/core/api/index.js:222:31)
    at Layer.handle [as handle_request] (/Users/robert/Dropbox/okp/node_modules/express/lib/router/layer.js:76:5)
    at next (/Users/robert/Dropbox/okp/node_modules/express/lib/router/route.js:100:13)
    at Route.dispatch (/Users/robert/Dropbox/okp/node_modules/express/lib/router/route.js:81:3)
    at Layer.handle [as handle_request] (/Users/robert/Dropbox/okp/node_modules/express/lib/router/layer.js:76:5)
    at /Users/robert/Dropbox/okp/node_modules/express/lib/router/index.js:227:24
    at Function.proto.process_params (/Users/robert/Dropbox/okp/node_modules/express/lib/router/index.js:305:12)
    at /Users/robert/Dropbox/okp/node_modules/express/lib/router/index.js:221:12
```

File: assets/css/project/info.less [hunk 1]
```diff
@@ -25,7 +31,6 @@
 		}
     }
 
-body p.source_info, /* cascade for weight */
```
Comment (operatino) [inline assets/css/project/info.less]: How did you get those changes in your branch? It was my refactoring from other branch.

**Comment by operatino** [comment]
And after turning on git dates, I get undefined http://d.pr/i/10fiu

**Comment by operatino** [comment]
I see you haven't merged your branch with 0.5.1-rc before making a PR.

File: assets/css/project/info.less (hunk 1 above)
Comment (cheshirsky) [inline assets/css/project/info.less]: It seems to be some merging artifact, I'm not sure why it's here. Shell I remove it?

**Comment by cheshirsky** [comment]
> And after turning on git dates, I get undefined from time to time http://d.pr/i/10fiu. Without specific errors in log.

I bet it's somehow connected with previous error stack trace. I'll try to find the cause of that asap. Can you provide your #ls -al result for me (in Skype)?

Both of that problems are quite tricky. I've tried to reproduce them last night, still nothing happens.

File: assets/css/project/info.less (hunk 1 above)
Comment (operatino) [inline assets/css/project/info.less]: Just merge your branch, and it will be fine.

**Comment by operatino** [comment]
PR should be re-opened to 0.5.1-dev branch.

**Pull request closed without merging**